examples/webrx/index.html

```diff
@@ -0,0 +1,79 @@
+﻿<!DOCTYPE html>
+
+<html data-framework="webrx" lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>WebRx - TodoMVC</title>
+    <link href="node_modules/todomvc-common/base.css" rel="stylesheet">
+    <link href="node_modules/todomvc-app-css/index.css" rel="stylesheet">
+</head>
+
+<body>
+    <section id="todoapp">
+        <header id="header">
+            <h1>todos</h1>
+            <input autofocus="" data-bind="textInput: @current, 
```

Comment:
No need for `=""` with empty attributes. Or is that needed by the framework somehow?